src/Poseidon/CLI/Forge.hs

```diff
@@ -255,7 +256,7 @@ filterSeqSourceRows (JannoRows jRows) (SeqSourceRows sRows) =
     where
         hasAPoseidonID :: [String] -> SeqSourceRow -> Bool
         hasAPoseidonID jIDs seqSourceRows =
```

Comment:
I guess that should be singular? `seqSourceRow` as the more correct variable name?

Reply:
True. Fixed :+1: